packages/phone-number-privacy/signer/src/common/metrics.ts

```diff
@@ -57,20 +57,48 @@ export const Counters = {
     help: 'Counter for the number of signature requests without a session id',
   }),
 }
+const buckets = [
```

Comment:
Would it be possible to rebase so that these changes from your other PRs aren't included?